apps/member-profile/app/routes/_profile.directory.tsx

```diff
@@ -204,9 +204,12 @@ export default function DirectoryPage() {
 
       <Dashboard.Subheader>
         <Dashboard.SearchForm placeholder="Search by name or email..." />
-
+        {/* <Dashboard.SearchForm placeholder="Search by school..." />
+        <Dashboard.SearchForm placeholder="Search by company..." /> */}
         <div className="ml-auto flex items-center gap-2">
-          <FilterDirectoryDropdown />
+          <FilterDirectoryDropdown filter='company' />
+          <FilterDirectoryDropdown filter='school' />
```

Comment:
I think tentatively these dropdowns look better if they are directly to the right of the search bar rather than all the way to the right. Perhaps we can move these two lines of code to right after line 206